_data/social.yml

```diff
@@ -249,7 +249,7 @@ websites:
       twitter: vanilla
 
     - name: Viadeo
-      url: http://www.viadeo.com
+      url: https://www.viadeo.com
```

Comment:
This site redirects to a HTTP version for me.
Is it just me or is anyone else getting this as well?